Specification.md

```diff
@@ -40,40 +40,78 @@ seq=11                                  \
 
 ## 2. Format and Naming
 
-**Sync Interest Name**: `/<group-prefix>/v=2/<parameters-digest>`
+**State Vector Data Name**: `/<group-prefix>/v=3`
 
-The State Vector is encoded in TLV format and included in the `ApplicationParameters` of the Sync Interest.
-The State Vector SHOULD be the first TLV block in the `ApplicationParameters`.
-The Interest Lifetime for Sync Interests is 1 second.
+The State Vector is encoded in the content of a signed NDN Data packet.
 
-**Data Interest Name**: `/<node-prefix>/<group-prefix>/<seq-num>`
+**Sync Interest Name**: `/<group-prefix>/v=3/<parameters-digest>`
+
+The State Vector Data is included in the `ApplicationParameters` of the Sync Interest.
+The Interest Lifetime for the Sync Interest is 1 second.
+
+**Data Interest name**: `/<node-prefix>/<group-prefix>/t=<bootstrap-time>/58=<seq>`
 
 _Note:_ Choosing alternative Data Interest formats may be decided on application-level.
 
-## 3. State Vector TLV Specification
+## 3. State Vector Specification
 
 ```abnf
 StateVector = STATE-VECTOR-TYPE TLV-LENGTH
               *StateVectorEntry
 
 StateVectorEntry = STATE-VECTOR-ENTRY-TYPE TLV-LENGTH
-                   NodeID
-                   SeqNo
+                   Name
+                   *SeqNoEntry
+
+SeqNoEntry = SEQ-NO-ENTRY-TYPE TLV-LENGTH
+             BootstrapTime
+             SeqNo
 
-NodeID = Name
+BootstrapTime = BOOTSTRAP-TIME-TYPE TLV-LENGTH NonNegativeInteger
 SeqNo = SEQ-NO-TYPE TLV-LENGTH NonNegativeInteger
 
 STATE-VECTOR-TYPE = 201
 STATE-VECTOR-ENTRY-TYPE = 202
-SEQ-NO-TYPE = 204
+
+SEQ-NO-ENTRY-TYPE = 210
+BOOTSTRAP-TIME-TYPE = 212
+SEQ-NO-TYPE = 214
 ```
 
-- The encoded state vector in the Interest consists of State Vector Entries
-- Each entry is a tuple of the NodeID of each node followed by its latest sequence number
-- The sequence number is 1-indexed, i.e. the first valid sequence number in a state vector is 1
+- The encoded state vector in the Interest consists of State Vector Entries.
+- Each entry has a node name followed by a list of [bootstrap timetamp, sequence number] tuples.
+- Bootstrap time is specified as seconds since the Unix epoch. Negative values are invalid.
```

Comment:
well, it's a NonNegativeInteger, so I'm not sure what "negative values" means...

Reply:
Making it explicit - if the system time is negative the app needs to throw an error.